frontend/www/js/omegaup/arena/contest_list.js

```diff
@@ -80,6 +80,8 @@ omegaup.arena.ContestList = function(element, apiParams, uiParams) {
             data.results.each(function(contest) {
               contest.contestLink = '/arena/' + contest.alias;
               contest.isVirtual = omegaup.UI.isVirtual(contest);
+              contest.scoreboardLink =
+                  contest.contestLink + '/scoreboard/' + contest.scoreboard_url;
```

Comment:
aquí puedes evitar el uso de `scoreboard_url` mediante

```suggestion
                  contest.contestLink + '#ranking'
```

Reply:
Ohhh esto funciona, pero de donde sale eso de **'#ranking'** ?

Reply:
de acá: https://github.com/omegaup/omegaup/blob/fabb3d5468cf8e5f5db44bbbc932227754ec4a71/frontend/templates/arena.contest.tpl#L13-L26

Reply:
Ooohhh, entonces removeré todo lo que agregué a algo distinto al **contest_list.js**